packages/renderer/src/components/molecules/AccountSidebar/_constants.ts

```diff
@@ -0,0 +1,56 @@
+import { LangKeys } from "@constants/lang";
+import { ROUTES } from "@constants/routes";
+import React from "react";
+import { useIntl } from "react-intl";
+
+export const WIDTH = 210;
+
+export interface AccountSidebarItem {
+  label: string;
+  route: string;
+}
+
+export const useGetAccountSidebarMenu = () => {
```

Comment:
The hook can be extracted out to a file in the same folder

Reply:
Maybe in `hooks.ts` file is much better.

Reply:
@schowdhuri Moved to `_hooks.ts` on same parent folder.